ota-plus-web/app/reactapp/src/stores/DevicesStore.js

```diff
@@ -542,10 +542,10 @@ export default class DevicesStore {
       );
   }
 
-  cancelApprovalPendingCampaingPerDevice(data) {
+  cancelApprovalPendingCampaingPerDevice(campaignId, deviceId) {
     resetAsync(this.mtuCancelAsync, true);
     return axios
-      .post(API_CANCEL_MULTI_TARGET_UPDATE, data)
+      .delete(`${API_CANCEL_UPDATE_PENDING_APPROVAL}/${campaignId}/devices/${deviceId}`)
```

Comment:
At first glance it seems you forgot to import `API_CANCEL_UPDATE_PENDING_APPROVAL` from `../config` 😉

Reply:
Thanks, that was it indeed :)